README.md

```diff
@@ -25,7 +25,9 @@ The list of available services at Attach can be found [here](http://gluonhq.com/
 
 Issues can be reported to the [Issue tracker](https://github.com/gluonhq/attach/issues)
 
-Contributions can be submitted via [Pull requests](https://github.com/gluonhq/attach/pulls)
+Contributions can be submitted via [Pull requests](https://github.com/gluonhq/attach/pulls), 
+ providing you have signed the [Gluon Individual Contributor License Agreement (CLA)](https://docs.google.com/forms/d/16aoFTmzs8lZTfiyrEm8YgMqMYaGQl0J8wA0VJE2LCCY) 
```

Comment:
s/providing/provided

Reply:
That's what we have in all the repos where we require the CLA.